files/epub.js/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "epub.js",
+  "repo": "futurepress/epub.js",
+  "files": {
+    "include": ["build/epub.min.js", "build/hooks.min.js", "build/reader.min.js", "build/libs/localforage.min.js", "build/libs/zip.min.js"]
```

Comment:
There is no build directory here https://github.com/futurepress/epub.js/tree/v0.3.2
Only dist/. I guess thats what you need to use